src/views/credits/credits.scss

```diff
@@ -1,39 +1,205 @@
 @import "../../colors";
 
-.credits {
-    p {
-        line-height: 1.5rem;
-
-        a {
-            white-space: normal;
-            word-wrap: break-word; /* Overrides: https://github.com/LLK/scratch-www/blob/develop/src/main.scss#L43-L47 */
-        }
-    }
+#view {
+    padding: 0;
+}
 
+.credits {
     ul {
         display: flex;
-        margin: 0;
-        padding: 0;
+        margin: 0 auto;
+        padding: 64px 0;
+        max-width: 864px;
         list-style: none;
         flex-wrap: wrap;
+        flex-flow: row wrap;
+        justify-content: center;
+    }
+
+    .content {
+        padding-top: 40px;
     }
 
     li {
         display: inline-block;
-        margin: 10px 0;
-        width: 188px;
-        text-align: center; 
+        margin: 0;
+        padding-bottom: 32px;
+        text-align: center;
+        line-height: 1.25rem;
 
         img {
-            margin: 10px 23px;
```

Comment:
Similar to the comment below about spans, you probably want to use a class name here. You don't want to rely on li > img "meaning" avatar, it makes it hard to change and makes it easy for others to make mistakes.

Reply:
In fact, this comment basically applies to any place where you are nesting an element selector underneath another element selector. Again, very happy to pair on this to fix!